lib/listings/useFetchCurrentUserListing.tsx

```diff
@@ -0,0 +1,12 @@
+import { useQuery } from "react-query";
+import ListingService from "./ListingService";
+import {
+    IBaseListing,
+    IListingAttributes,
+    TListingCollectionReference,
+    TListingDocumentReference,
+  } from "./types"
+
+export default function useFetchCurrentUserListing(uid: any) {
```

Comment:
we should name this input userId to be explicit and define it as a string type instead of a any

Reply:
changed it to TListingCollectionReference because that is the type being used for listing service